### PR TITLE
W-14414918: Chore/cherrypicks 20231122 

### PR DESCRIPTION
**Pull request by VTorrazza**
Ensure use of on org.apache.santuario 2.3.4 to fix vulnerability

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -221,6 +222,16 @@
 
     </dependencies>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.apache.santuario</groupId>
+                <artifactId>xmlsec</artifactId>
+                <version>2.3.4</version>
```
Comment (juanfloresMule) [inline pom.xml:230]: Can we use a property for the version?

File: pom.xml [hunk 2]
```diff
@@ -37,14 +38,14 @@
         <easyrandom.version>4.0.0</easyrandom.version>
         <mule.wss.module.version>1.1.2</mule.wss.module.version>
         <bcprov-jdk15on.version>1.70</bcprov-jdk15on.version>
-        <mule.file.connector.version>1.5.0</mule.file.connector.version>
+        <mule.file.connector.version>1.5.1</mule.file.connector.version>
```
Comment (sabecasismulesoft) [inline pom.xml:41]: This was not part of your original changes, why did you cerry-pick this? Is it affecting the tests only or the connector compile functionality?

File: pom.xml (hunk 1 above)
Comment (VTorrazza) [reply pom.xml:230]: sure, I'll change it

File: pom.xml (hunk 2 above)
Comment (VTorrazza) [reply pom.xml:41]: I merged renovate PR's and this was one of the changes suggested, should I remove it?

Comment (VTorrazza) [reply pom.xml:41]: I did cherry-pick of my changes and the renovate's PR

File: pom.xml (hunk 1 above)
Comment (VTorrazza) [reply pom.xml]: done

**Review by sabecasismulesoft — APPROVED** [review]

**Pull request merged**